integration/client/client_darwin_test.go

```diff
@@ -0,0 +1,43 @@
+// +build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+var (
+	defaultRoot    = "/var/lib/containerd-test"
+	defaultState   = "/var/run/containerd-test"
+	defaultAddress = "/var/run/containerd-test/containerd.sock"
+	testImage      string
+	shortCommand   = withProcessArgs("hello")
+	longCommand    = withProcessArgs("ping", "127.0.0.1")
+)
+
+func init() {
+	testImage = "docker.io/ukontainer/runu-base:0.3-osx-extra"
```

Comment:
Can we pass the image string via GitHub Actions yaml?

I don't think we have reached consensus on having runU dependency on Go codes. (Neither on yamls, but yamls might be easier than Go codes to reach consensus)

Reply:
Also,  we want to avoid using Docker Hub, unless they allow limitless pulling for GHA macOS instances.

Reply:
> Also, we want to avoid using Docker Hub, unless they allow limitless pulling for GHA macOS instances.

i'm not familiar with alternatives.  do you have any recommendation for an alternate repository ?

Reply:
ghcr.io seems good

Reply:
thanks, fixed as you suggested, using ghcr instead of docker hub.